### PR TITLE
Embargoes logs

### PR DESCRIPTION
**Pull request by qadan**

### CODE REVIEW

File: embargoes.module [hunk 1]
```diff
@@ -56,9 +56,21 @@ function embargoes_theme($existing, $type, $theme, $path) {
...
+          [
+            'label' => NULL,
+            'proxy_url' => NULL,
+          ],
+        ],
         'contact_email' => NULL,
       ],
     ],
+    'embargoes_policies' => [
+      'template' => 'embargoes-policies',
```
Comment (IAMlKeno) [inline embargoes.module:69]: does this template exists?

File: src/Form/EmbargoesEmbargoEntityForm.php [hunk 2]
```diff
@@ -184,13 +191,15 @@ public function save(array $form, FormStateInterface $form_state) {
...
 
     if ($status == SAVED_NEW) {
-      $log_values['action'] = 'created';
+      $log_values['action'] = $embargo::STATUS_CREATED;
+      $action = 'created';
     }
     else {
-      $log_values['action'] = 'updated';
+      $log_values['action'] = $embargo::STATUS_UPDATED;
+      $action = 'updated';
```
Comment (IAMlKeno) [inline src/Form/EmbargoesEmbargoEntityForm.php]: probably could have made these constants as well?

File: embargoes.module (hunk 1 above)
Comment (qadan) [reply embargoes.module:69]: i think i likely just forgot to add this guy

File: src/Entity/EmbargoesEmbargoEntity.php [hunk 3]
```diff
@@ -262,12 +246,38 @@ public function getNotificationStatus() {
...
+    }
     $this->set('notification_status', $status);
     return $this;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTagsToInvalidate() {
+    $tags = parent::getCacheTagsToInvalidate[];
```
Comment (JojoVes) [inline src/Entity/EmbargoesEmbargoEntity.php]: should be () not []?

File: embargoes.install [hunk 4]
```diff
@@ -56,3 +61,18 @@ function embargoes_update_8001(&$sandbox) {
...
+ * Update the 'action' column to use integers. Add indices.
+ */
+function embargoes_update_8002(&$sandbox) {
+  $schema = Database::getConnection()->schema();
+  $schema->changeField('embargoes_log', 'action', 'action', [
+    'type' => 'int',
+    'size' => 'tiny',
+    'not null' => TRUE,
+  ]);
+  $schema->addIndex('embargoes_log', 'action', ['action']);
```
Comment (JojoVes) [inline embargoes.install]: `[error]  Too few arguments to function Drupal\Core\Database\Driver\pgsql\Schema::addIndex(), 3 passed in /opt/www/drupal/modules/contrib/embargoes/embargoes.install on line 75 and exactly 4 expected`

**Pull request merged**